src/main/java/parser/Parser.java

```diff
@@ -126,8 +129,11 @@ public Command unmark(String input) throws LuluException {
         if (words.length <= 1) {
             throw new InvalidCommandException();
```

Comment:
Uh you might have forgotten to put throws InvalidCommandException in your function descriptor next to your LuluException? Since you don't catch it in this function :) Same for some other functions in this file as well
The javadocs would have another line for that too! 😄